packages/dashboard/src/pages/user/settings.js

```diff
@@ -0,0 +1,85 @@
+import * as Yup from "yup";
+import Layout from "../../components/Layout";
+import SelfServiceForm from "../../components/Form/SelfServiceForm";
+import accountsApi from "../../services/accountsApi";
+
+const profileConfig = [
+  {
+    name: "email",
+    type: "text",
+    label: "New email address",
+    autoComplete: "email",
+    position: 0,
+  },
+  {
+    name: "confirmEmail",
+    type: "text",
+    label: "New email address repeated",
+    autoComplete: "email",
+    position: 1,
+  },
+];
+
+const passwordConfig = [
+  {
+    name: "password",
+    type: "password",
+    label: "New password",
+    position: 1,
+  },
+  {
+    name: "confirmPassword",
+    type: "password",
+    label: "New password repeated",
+    position: 2,
+  },
+];
+
+const emailValidationSchema = Yup.object().shape({
+  email: Yup.string().required("Email is required").email("This email is invalid"),
+  confirmEmail: Yup.string().oneOf([Yup.ref("email"), null], "Emails must match"),
+});
+
+const passwordValidationSchema = Yup.object().shape({
+  password: Yup.string().required("Password is required").min(6, "Password has to be at least 6 characters long"),
+  confirmPassword: Yup.string().oneOf([Yup.ref("password"), null], "Passwords must match"),
```

Comment:
Am I reading this correctly, as the `confirmPassword` being allowed to be `null`?
I see other instances of this but I won't duplicate the same comment there.